_translator-files/po/es/Client-Troubleshooting.po

```diff
@@ -225,24 +214,18 @@ msgstr "Puedes establecer el \"Nivel Cliente Nuevo\" a un valor bajo (por ej. 10
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:72
-#, fuzzy
-#| msgid "### Can't see the server you want to join?"
 msgid "### Can't see the Server you want to join?"
-msgstr "### ¿No ves el servidor al que te quieres unir?"
+msgstr "### ¿No ves el Servidor al que te quieres unir?"
 
 #. type: Plain text
 #: ../wiki/en/Client-Troubleshooting.md:74
-#, fuzzy
-#| msgid "First check that you have the right genre server selected in your Connection Setup window. Sometimes network issues mean your client won't list all the available servers. If you know the name of the server you want to join, you can [look up its IP address here](https://explorer.jamulus.io/). Enter the IP address in the \"Server Name/Address\" field in the Connect Setup window to connect to it."
 msgid "First check that you have the right genre Server selected in your Connect window. Sometimes network issues mean your Client won't list all the available Servers. If you know the name of the Server you want to join, you can [look up its IP address here](https://explorer.jamulus.io/). Enter that address in the \"Server Address\" field in the Connect Setup window to connect to it."
-msgstr "Primero comprueba que tienes el género de servidor correcto seleccionado en la ventana de Configuración de Conexión. En ocasiones los problemas de red hacen que tu cliente no muestre todos los servidores disponibles. Si conoces el nombre del servidor al que te quieres unir, puedes [mirar su dirección IP aquí](https://explorer.jamulus.io/). Introduce la dirección IP en el campo de \"Nombre/Dirección Servidor\" en la ventana de Configuración de Conexión para conectarte."
+msgstr "Primero comprueba que tienes el género de Servidor correcto seleccionado en la ventana de Conectar. En ocasiones los problemas de red hacen que tu Cliente no muestre todos los Servidores disponibles. Si conoces el nombre del Servidor al que te quieres unir, puedes [mirar su dirección IP aquí](https://explorer.jamulus.io/). Introduce esa dirección en el campo de \"Dirección Servidor\" en la ventana de Configuración de Conexión para conectarte."
```

Comment:
`Introduce esa dirección en el campo de \"Dirección Servidor\" en la ventana de Configuración de Conexión para conectarte."`
This is wrong.  The field being described is the one at the bottom of the Connect dialogue where you enter the address of the Server to which you want to connect.  Nothing to do with Directories (no "server" when mentioning Directories).

Reply:
There's no mention of "directory" here. "Dirección" means "address".

Reply:
Ah yes... that _is_ the label for "Server address" in English :).